tests/core/test_test.py

```diff
@@ -1236,12 +1236,12 @@ def test_freeze_time(mocker: MockerFixture) -> None:
 
     @model("py_model", columns={"ts1": "timestamptz", "ts2": "timestamptz"})
     def execute(context, start, end, execution_time, **kwargs):
-        datetime_now = datetime.datetime.now()
+        datetime_now_utc = datetime.datetime.now(datetime.UTC)
```

Comment:
The `datetime.now` function [produces a local datetime by default](https://docs.python.org/3/library/datetime.html#datetime.datetime.now), so the python model test here was failing with `time-travel`, whereas `freezegun` worked because it patched it to `FakeDatetime(2023, 1, 1, 10, 5, 3)`.

cc @treysp